server/src/app.ts

```diff
@@ -47,6 +60,11 @@ const identityClient =
 app.get('/api/config', configController(config));
 app.get('/api/token', tokenController(identityClient, config));
 
+const cosmosClient = new CosmosClient(config);
+const surveyDBHandler = new SurveyDBHandler(cosmosClient);
+
+app.post('/api/createSurveyResult', storeSurveyResult(surveyDBHandler));
```

Comment:
Make this `POST /api/surveyResults` maybe?

Reply:
Since it is post method, I think that make sense! I will update the code.

Reply:
What do you think about `surveyResult` since it is only handling to store one request per api call ?

Reply:
Actually I am going with `api/surveyResults` 😄